.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+## Overview
+**TL;DR**
+<One to two sentence description of the issue you are encountering or trying to solve.>
+
+### Extra Detail
+
+#### Screenshots
+<If bug fix, please upload images of the error to better showcase the problem.>
+
+#### Reasoning
+<If new idea or direction, please create a checklist of proposed ideas or next steps. [Example](https://github.com/github/training-kit/issues/415).>
+
+### How to Contribute
+- [ ] Comment on this issue if you'd like to work/collaborate on it! One of the training-kit maintainers will then "Assign" you to the issue.
```

Comment:
@hollenberry we can't assign people who do not have collaborator access to the repo.